### PR TITLE
fix kserve test scripts

### PR DESCRIPTION
**Pull request by dtrifiro**
`deploy-model-http.sh` and `deploy-model-grpc.sh` now call `deploy-model.sh` with the appropriate argument

### CODE REVIEW

**Comment by dtrifiro** [comment]
Smoke test failure is unrelated to this change.

**Comment by dtrifiro** [comment]
CI failures fixed by #219

File: demo/kserve/scripts/test/grpc-console.sh [hunk 1]
```diff
@@ -18,12 +18,12 @@ echo "Wait until runtime is READY"
 wait_for_pods_ready "serving.kserve.io/inferenceservice=caikit-tgis-isvc-grpc" "${TEST_NS}"
 oc wait --for=condition=ready pod -l serving.kserve.io/inferenceservice=caikit-tgis-isvc-grpc -n ${TEST_NS} --timeout=300s
 
-export KSVC_HOSTNAME=$(oc get ksvc caikit-tgis-isvc-grpc-predictor -n ${TEST_NS} -o jsonpath='{.status.url}' | cut -d'/' -f3)
+export ISVC_HOSTNAME=$(oc get ksvc caikit-tgis-isvc-grpc-predictor -n ${TEST_NS} -o jsonpath='{.status.url}' | cut -d'/' -f3)
```
Comment (Xaenalt) [inline demo/kserve/scripts/test/grpc-console.sh:21]: Should get from ISVC

File: demo/kserve/scripts/test/http-call.sh [hunk 2]
```diff
@@ -23,15 +22,14 @@ echo
 echo "Testing all token in a single call"
 echo
 
-export KSVC_HOSTNAME=$(oc get ksvc "${ISVC_NAME}"-predictor -n ${TEST_NS} -o jsonpath='{.status.url}' | cut -d'/' -f3)
+ISVC_URL=$(oc get ksvc "${ISVC_NAME}"-predictor -n ${TEST_NS} -o jsonpath='{.status.url}')
```
Comment (Xaenalt) [inline demo/kserve/scripts/test/http-call.sh:25]: Should get from ISVC

**Review by Xaenalt — CHANGES_REQUESTED** [review]

File: demo/kserve/scripts/test/grpc-console.sh (hunk 1 above)
Comment (dtrifiro) [reply demo/kserve/scripts/test/grpc-console.sh]: done

File: demo/kserve/scripts/test/http-call.sh (hunk 2 above)
Comment (dtrifiro) [reply demo/kserve/scripts/test/http-call.sh]: done

**Review by Xaenalt — APPROVED** [review]

**Pull request merged**